gem/context_processors.py

```diff
@@ -48,21 +49,23 @@ def compress_settings(request):
         if oidc_settings:
             REGISTRATION_URL = (
                 "%s/registration/?theme=%s&hide=end-user&redirect_uri=%s"
-                "&client_id=%s" % (
+                "&client_id=%s&language=%s" % (
                     settings.OIDC_OP, settings.THEME,
                     request.site.root_url
                     + reverse('oidc_authentication_init'),
-                    oidc_settings.oidc_rp_client_id))
+                    oidc_settings.oidc_rp_client_id, language))
             VIEW_PROFILE_URL = (
-                "%s/profile/edit/?theme=%s&redirect_uri=%s&client_id=%s" % (
+                "%s/profile/edit/?theme=%s&redirect_uri=%s&client_id=%s"
+                "&language=%s" % (
                     settings.OIDC_OP, settings.THEME,
                     oidc_settings.wagtail_redirect_url,
-                    oidc_settings.oidc_rp_client_id))
+                    oidc_settings.oidc_rp_client_id, language))
             EDIT_PROFILE_URL = (
-                "%s/profile/edit/?theme=%s&redirect_uri=%s&client_id=%s" % (
+                "%s/profile/edit/?theme=%s&redirect_uri=%s&client_id=%s"
+                "&language=%s" % (
                     settings.OIDC_OP, settings.THEME,
                     oidc_settings.wagtail_redirect_url,
-                    oidc_settings.oidc_rp_client_id))
+                    oidc_settings.oidc_rp_client_id, language))
```

Comment:
what about login and log out?

Reply:
Login is handled differently and the code added to views.py covers that.
The logout link doesn't take us to any screens on the auth service so I don't think it's necessary

Reply:
okay cool

Reply:
@codiebeulaine can you confirm that my comment above about the logout link is correct?